contracts/script/deploy/eigenda-directory/config/example.config.toml

```diff
@@ -1,12 +1,8 @@
-
-# This field should be populated with the contract names for use in contract lookups.
-contractNames = ["EXAMPLE", "EXAMPLE2"]
-
-# This field should be populated with the addresses of the contracts in the same order as in contractNames.
-contractAddresses = ["0xfC4C575ea3f8316D38F83076Cf765f7B8DeaFC58", "0xF2d1511E53AeAE16213EcA7963A103cB5CEc2b18"]
-
 # The address of the proxy admin contract that manages the upgradeability of the contracts.
-proxyAdmin = "0x9Fd7E279f5bD692Dc04792151E14Ad814FC60eC1"
+proxyAdmin = "0xF36fDAf681E3D88Ee412879aa4907B60D6ff0707"
 
 # The privileged account that can manage the contract directory.
-owner = "0xDA29BB71669f46F2a779b4b62f03644A84eE3479"
\ No newline at end of file
+owner = "0xF36fDAf681E3D88Ee412879aa4907B60D6ff0707"
+
+[contracts]
+EXAMPLE = "0xF36fDAf681E3D88Ee412879aa4907B60D6ff0707"
```

Comment:
Guessing this is just a random example address? 
Why not use 0x0, 0x1, 0x2 or something? I was puzzled looking at this for a minute, wondering why proxyAdmin and owner were the same address, until I realized EXAMPLE is the same.

Reply:
Yes, I like to just use newly generated keys as example keys since I can be reasonably sure they aren't being used.

Reply:
It's not really the case here, but there are scenarios in which using a known address can be very problematic especially if those addresses have any funds. If an example script were to use a known address and be used in tests, the fact that an account has funds may be missed. 

So I like to avoid using any commonly known addresses. The quickest way I know to get a clean address that nobody has access to is to generate a new one in my terminal.